includes/Autopromote.php

```diff
@@ -28,45 +28,6 @@ public static function getAutopromoteGroups( User $user ) {
 	}
 
 	/**
-	 * Get the groups for the given user based on the given criteria.
```

Comment:
Yes, autopromote per se is still used. What is not used is AutopromoteONCE - that is a subset of the whole autopromote logic.